.circleci/config.yml

```diff
@@ -128,4 +128,8 @@ workflows:
       - fmt
       - clippy
       - doc
-      - build
+      - build:
```

Comment:
No that was available before. The rationale is that using a VM is slower than a container, so I want to avoid building before the other passes have returned successfully.

Reply:
That makes sense. But why then don't we keep using containers for things that won't run tests, like fmt, clippy, etc?

Reply:
Done, we now use Docker for all tasks except the build one